recipes/kit4dl/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "kit4dl" %}
+{% set version = "2023.08b0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  path: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/kit4dl-{{ version }}.tar.gz
+  sha256: 7ab8579afdf042b32559b9cc8eac72ed34d3dbd1fba87e0dfa09353cf7d65ce4
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - kit4dl = kit4dl.cli.app:run
+
+requirements:
+  host:
+    - python >=3.10
+    - pip    
+  run:
+    - python >=3.10
+    - pytorch
+    - lightning
+    - torchmetrics
+    - numpy >=1.23.0
+    - pydantic >=1.10.0
+    - scikit-learn >=1.2.0
+    - colormath
+    - tqdm
+    - colorama >=0.4.3,<0.5.0
+    - shellingham >=1.3.0,<2.0.0
+    - rich >=10.11.0,<14.0.0
+    - typer
```

Metadata has toml there.

```suggestion
    - typer
    - toml
```

---

yes, but it's needed for python 3.10. for python 3.11 there is builtint library `tomlib`. anyway , i can add this.